frontend/www/js/omegaup/problem/collection_list.ts

```diff
@@ -58,17 +65,23 @@ OmegaUp.on('ready', () => {
           tagsList: payload.tagsList,
           sortOrder: sortOrder,
           columnName: columnName,
+          difficulty: payload.difficulty,
```

Comment:
El error que está marcando GitHub tiene que ver con que estás asignando un valor a la variable `difficulty`, sin embargo, estás usando el que viene directo del `payload`. La solución puede ser la siguiente:  
```suggestion
          difficulty: difficulty,
```